### PR TITLE
DROOLS-3903 DMN Submarine endpoint codegen

### PR DESCRIPTION
**Pull request by tarilabs**
requires https://github.com/kiegroup/kogito-runtimes/pull/121

### CODE REVIEW

File: dmn-quarkus-example/src/test/java/org/kie/dmn/kogito/quarkus/example/TrafficViolationTest.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                     "    },\n" +
+                     "    \"Violation\": {\n" +
+                     "        \"Type\": \"speed\",\n" +
+                     "        \"Actual Speed\": 120,\n" +
+                     "        \"Speed Limit\": 100\n" +
+                     "    }\n" +
+                     "}")
+               .contentType(ContentType.JSON)
+          .when()
+               .post("/Traffic Violation")
```
Comment (tarilabs) [inline dmn-quarkus-example/src/test/java/org/kie/dmn/kogito/quarkus/example/TrafficViolationTest.java:47]: The pertinent DMN REST resources is now generated ad-hoc for the model, and the model name is `Traffic Violation`

File: dmn-quarkus-example/src/test/java/org/kie/dmn/kogito/quarkus/example/TrafficViolationTest.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import static org.hamcrest.Matchers.hasItem;
+import static org.hamcrest.Matchers.is;
+
+@QuarkusTest
+public class TrafficViolationTest {
+    
+    @Test
+    public void testEvaluateTrafficViolation() {
+        given()
+               .body("{\n" +
```
Comment (tarilabs) [inline dmn-quarkus-example/src/test/java/org/kie/dmn/kogito/quarkus/example/TrafficViolationTest.java:35]: atm the payload is not typed, ref. roadmap.

**Comment by tarilabs** [comment]
doesn't Jenkins here pick up the same branch name on the kogito-runtimes?
(see linked PR)

**Comment by tarilabs** [comment]
I am getting

```
[ERROR]   TrafficViolationTest.testEvaluateTrafficViolation:49 1 expectation failed.
Expected status code <200> but was <405>.
```

which makes me wonder if this Jenkins instance allows me to perform rest tests?

**Comment by mswiderski** [comment]
@tarilabs yes, you can use rest tests in Jenkins without issues, see this one that runs on each build
https://github.com/kiegroup/kogito-examples/blob/master/jbpm-quarkus-example/src/test/java/org/kie/kogito/examples/OrdersRestTest.java

**Comment by tarilabs** [comment]
weird, as it works locally just fine, only on Jenkins I get a 405 ?

**Comment by tarilabs** [comment]
image

**Comment by tarilabs** [comment]
could be Jenkins did not reckon the old name of my fork. I updated my forks name, let's see if that solves..

**Comment by tarilabs** [comment]
that solved.

**Comment by tarilabs** [comment]
this was CI based on https://github.com/kiegroup/kogito-runtimes/pull/125 as expected

**Pull request merged**